defaults/main.yml

```diff
@@ -119,6 +122,7 @@ postgresql_ssl_ecdh_curve: 'prime256v1'
 postgresql_ssl_renegotiation_limit: 512MB # amount of data between renegotiations
 postgresql_ssl_cert_file: /etc/ssl/certs/ssl-cert-snakeoil.pem  # (>= 9.2)
 postgresql_ssl_key_file: /etc/ssl/private/ssl-cert-snakeoil.key # (>= 9.2)
+postgresql_ssl_csr_file: ''
```

Comment:
The CSR is really only for use temporarily... but I have kept it as a permanent file so that Ansible doesn't register it as "Changed" each time.